### ISSUE TITLE
Accessibility: Remove H3 (header 3) in various locations

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#1400]
When user is depositing an item, on the 'Title and contact' page, the 'Keywords' section is tagged as an H3, and it should not be.

Also:
Edit item
* Author/Contributors: There is an H3 tag around each of the forms where user enters details about each author/contributor.
* Abstract/Keywords: H3 tag around Keywords
* Dates: H3 tag on Publication date and around Y/M/D fields under creation date

Edit collection
* Details: H3 around Contact email
* Participants: H3 around Managers and Depositors
* Contributors: H3 around forms where user enters details about each contributor

**Comment by astridu** [comment]
Here's a screen recording of Voiceover (you can see the dialogue box in the lower left, but it doesn't have the audio). You can see as I toggle through H1, H2, and then there's an H3 which is a whole section, and that section should not be tagged as an H3.
https://github.com/user-attachments/assets/e5adbba8-eed4-4831-90b8-435499f2e066

Here is a snippet of the code:

<img width="402" alt="Image" src="https://github.com/user-attachments/assets/49a13e44-fa7e-4796-8bcc-a125ca7f80d7" />
